official/boosted_trees_upgraded/README.md

```diff
@@ -0,0 +1,112 @@
+# Classifying Higgs boson processes in the HIGGS Data Set
```

Comment:
High level question (for @tfboyd  as well): why don't we just make sure this model is 1.x and 2.0 compatible? The major changes that I see needed is to have HIGGS dataset as our new tfds. (Assuming 1.x supports tfds)

Reply:
One set of code is fine.  We have two sets in situations where we moved from layers to keras or estimator to keras.fit.  If that is not the case here then just upgrade and remove contrib.

Reply:
I also agree with @tfboyd.